.github/workflows/cypress.yaml

```diff
@@ -2,9 +2,9 @@ name: Cypress Test
 on:
   workflow_call:
     secrets:
-      CYPRESS_PPC_AWS_ACCESS_KEY_ID:
+      CYPRESS_USER_EMAIL:
```

Comment:
There is no need to keep both of our new variables in secrets. The email should be stored in vars, not in secrets, so we can see which user is being used. The password can remain in secrets.

Reply:
you are correct, I already have email in variables and not in secrets so I will change it here.